lib/bootscale/file_storage.rb

```diff
@@ -1,4 +1,5 @@
 require 'digest/md5'
+require 'active_support/core_ext/file'
```

Comment:
Unfortunately we can't use Active Support's implementation, because it's not one of bootscale dependencies, and bootscale can be used for non-rails project.

You can just copy the implementation in a `Bootscale::Utils` module or something.

Also I suggested Sprocket's one because it's simpler. AS does a few things around file access mode etc that we don't need.